docs/run-instance/installation/running-jar-war.md

```diff
@@ -0,0 +1,70 @@
+# Running JAR / WAR
+
+## Provided artifacts
+
+For release version 8.0.0 and newer, we provide JAR and WAR files as artifacts for download. The JAR file can be run stand-alone and will start openrouteservice using a built-in tomcat server. Alternatively, you can set up your own tomcat instance on your server and deploy the WAR artifact on it.  
+
+## Parameters and environment variables
+
+Openrouteservice is configured by pointing an environment variable called `ORS_CONFIG_LOCATION` to a YAML configuration file. Use the following commands to download our example configuration file (which is used by our docker setup by default) to have a starting point, and export the path to that file to the required environment variable.
+
+```shell
+wget https://raw.githubusercontent.com/GIScience/openrouteservice/master/ors-api/ors-config.yml
+export ORS_CONFIG_LOCATION=${pwd}/ors-config.yml
+```
+
+For more details on the configuration options with that YAML file, see the chapter on [configuration](/run-instance/configuration/).
+
+If you already have an old and deprecated JSON format configuration file, you can still use that configuration file by setting an environment variable called `ORS_CONFIG` to point that file. Currently, all settings in a provided JSON configuration file will *override* any settings in the YAML file. We strongly recommend to migrate your settings to the new YAML format though, since we are planning to remove the support for JSON configuration files with the next major version release.  
+
+## Folders and files
+
+Openrouteservice produces output files of three types, for which the paths can be configured. The directories these paths point to need to be *writable*. The default configuration file for openrouteservice mentioned above provides the following paths:
+- `./ors-core/data/graphs`: The converted graphs for the profiles are stored here.
+- `./ors-core/data/elevation_cache`: If elevation is activated in the configuration, openrouteservice will download and cache the elevation data tiles here. 
+- `./logs`: Log output is written to auto-rotated log files. 
+ 
+Openrouteservice also requires an OpenStreetMap export file to import the graph data from, configured in the YAML configuration file. The default location is `./ors-core/data/osm_file.pbf`. 
+
+See chapter [logging](/run-instance/configuration/logging) for details on configuring the location of log files, and chapter [engine](/run-instance/configuration/engine) for the all other file locations.
+
+[//]: # (TODO: overhaul contents below after integrating the jar build PR)
+
+# Installing and running tomcat9
```

Comment:
Tomcat 10, no?

Reply:
sure, but like the comment says this document is WIP and needs overhaul after we finally integrate the JAR build feature...

Reply:
Relating the Jar yes. But if someone opens this doc and tries to start ors with latest master and Tomcat 9, it'll fail due to the latest spring update. Am I mixing something up?

Reply:
Tomcat 10, on the other hand, should be downwards compatible

Reply:
I've updated the file to contain no wrong information.